common/frontend/utils.js

```diff
@@ -38,16 +38,22 @@ export async function onSubmit(event) {
   fieldset.disabled = true
 
   // make an update call to the smart contract
-  await contract.setGreeting({
```

Comment:
@chadoh why change from `async` / `await` to returning promise explicitly?

Reply:
...because I hadn't known about `try...catch...finally`

I've updated to use this syntax, thanks for teaching me!